config/jobs/kubernetes-sigs/image-builder/image-builder-presubmits.yaml

```diff
@@ -4,7 +4,7 @@ presubmits:
     labels:
       preset-azure-cred: "true"
     decorate: true
-    run_if_changed: 'images/capi/.*'
+    run_if_changed: 'images/capi/(Makefile|ansible\.cfg)|images/capi/ansible/.*|images/capi/scripts/ci-azure-e2e\.sh|images/capi/packer/(config|goss|azure)/.*|images/capi/hack/ensure-(ansible|packer|goss).*'
```

Comment:
Does `ensure-ansible-windows.sh` match in this regex?
Will prob need it in ova after we enable windows ova ci.

Reply:
It does, yes. it's essentially saying `images/capi/hack/ensure-ansible*`, which matches both `ensure-ansible.sh` and `ensure-ansible-windows.sh`.